### DISCUSSION TITLE
Future of termux

### DISCUSSION THREAD

**Discussion by C9-hp** [termux/termux-app#3710] [General]
As you know termux had some problems with android 10 i think. So in the future new updates for Android are coming. 

Bot termux is still made for Android 8 ? There is no way to update it?

**Comment by sylirre** [comment]
Termux force enables compatibility mode with Android 9. This doesn't mean it is built for Android 8 or 9.

This will be changed eventually but not a thing to worry about now.

**Reply by C9-hp** [reply]
Right
